src/test/java/baron/Utils/FileUtilsTest.java

```diff
@@ -0,0 +1,26 @@
+package baron.Utils;
+
+import org.junit.jupiter.api.Test;
+
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class FileUtilsTest {
+    public static final String BASE_DIR = System.getProperty("user.dir") + "/src/test/java/baron.Database";
+
+    public FileUtilsTest() {
+
+    }
```

Comment:
I like how you have added an empty constructor since it may not be apparent to people who are new to Java that every class would be coupled with a default empty constructor 💯